var/spack/repos/builtin/packages/py-joeynmt/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyJoeynmt(PythonPackage):
+    """Minimalist NMT for educational purposes"""
+
+    homepage = "https://https://github.com/joeynmt/joeynmt"
+    git = "https://github.com/joeynmt/joeynmt.git"
```

Comment:
Can we use stable archive URLs instead? It would also be good to add a comment saying we aren't using the PyPI URLs because the sdist is broken (requirements.txt is missing).